### DISCUSSION TITLE
Is it possible to send no tests found message to slack

### DISCUSSION THREAD

**Discussion by AptecoJoe** [ryanrosello-og/playwright-slack-report#186] [Q&A]
Sometimes the pipeline fails its set up so the tests aren't run.

The following is reported: 'Slack reporter - Playwright reported : "No tests found"'

Is it possible for me to have this message sent to the slack channel?

**Comment by ryanrosello-og** [comment]
GH issue created -> https://github.com/ryanrosello-og/playwright-slack-report/issues/230
